### DISCUSSION TITLE
no updates

### DISCUSSION THREAD

**Discussion by Tricked-dev** [NOperatingSystem/NOperatingSystem#26] [General]
smh my who update mnan

**Comment by mugman174** [comment]
Yea it’s sad. @jdadonut is supposed to be working on entering 32 bit protected mode but it doesn’t work. If you/anyone know(s) pls PR.

**Comment by mugman174** [comment]
If you want, we have a discord [here](https://discord.gg/KaQJ6pqrPF).

**Comment by Tricked-dev** [comment]
ill make a pr brb

**Comment by mugman174** [comment]
It's still not entering 32-bit mode from what I see...
![Booting from Hard Disk...
noOS: Started in 16-bit real mode, Loading...](https://user-images.githubusercontent.com/53713683/108094242-75630a00-7033-11eb-87e0-a8bc0273b141.png)

**Comment by aenriii** [comment]
p

> 
> 
> Yea it’s sad. @jdadonut is supposed to be working on entering 32 bit protected mode but it doesn’t work. If you/anyone know(s) pls PR.

yeah, every method ive seen to get into 32bpm just bootloops the emulator

**Comment by mugman174** [comment]
> yeah, every method ive seen to get into 32bpm just bootloops the emulator
Anyone who can help, pls help us with this.

**Comment by aenriii** [comment]
idea: convert to efi booting which *should* automagically put us in 64 bit long mode

**Comment by aenriii** [comment]
gonna try this locally, if it works i'll open a pull request

**Comment by aenriii** [comment]
using zig
